abdm_integrator/utils.py

```diff
@@ -115,3 +117,19 @@ class APIResultsSetPagination(PageNumberPagination):
     page_size = 10
     page_size_query_param = 'page_size'
     max_page_size = 1000
+
+
+def poll_for_data_in_cache(cache_key, total_attempts=30, interval=2):
+    attempt = 1
+    while attempt <= total_attempts:
+        time.sleep(interval)
+        data = cache.get(cache_key)
```

Comment:
It's best to manage all of cache.set/get/delete in a single utility class and use the class as an interface everywhere else.

Reply:
Addressed in 93fce5df4d1cf82959eecb6d8d2b7608138ee215